rel/overlay/etc/default.ini

```diff
@@ -512,6 +512,40 @@ min_priority = 2.0
 [smoosh.ratio_views]
 min_priority = 2.0
 
+[ioq]
+; The maximum number of concurrent in-flight IO requests that
```

Comment:
remove "that"?

Reply:
Good catch, will address on the next change to the .ini file